### PR TITLE
[KIP-847] Util to upload Checkpoint

### PR DESCRIPTION
**Pull request by giangtong-katalon**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 setuptools.setup(
     name="testops-commons",
-    version="1.0.1",
+    version="1.0.5",
```
Comment (xuanan-gy) [inline setup.py]: consider for 1.1.0 because the third number is for the bug fixing version, second for minor feature, first for the major feature

**Review by xuanan-gy — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (giangtong-katalon) [reply setup.py]: removed. Only update version when merge release

Comment (giangtong-katalon) [reply setup.py]: Will consider the version number later

**Review by xuanan-gy — APPROVED** [review]

**Review by nguyen-tri-nhan — APPROVED** [review]

File: testops_commons/uploader/uploader.py [hunk 2]
```diff
@@ -1,17 +1,24 @@
+import json
+import logging
 from os import path
```
Comment (tuananht) [inline testops_commons/uploader/uploader.py]: from os import getenv, path

File: testops_commons/core/constants.py [hunk 3]
```diff
@@ -43,3 +43,8 @@
 TESTOPS_PROXY_PASSWORD_ENV = 'TESTOPS_PROXY_PASSWORD'
 
 REPORT_PATTERN = r'.*(?i)\.json'
+
+KATALON_TESTOPS_PROJECT_ID_ENV = 'KATALON_TESTOPS_PROJECT_ID'
+KATALON_TESTOPS_API_KEY_ENV = 'KATALON_TESTOPS_API_KEY'
+KATALON_TESTOPS_SERVER_ENV = 'KATALON_TESTOPS_SERVER'
+KATALON_TESTOPS_SESSION_ID_ENV = 'KATALON_TESTOPS_SESSION_ID'
```
Comment (tuananht) [inline testops_commons/core/constants.py]: TESTOPS_....

**Review by tuananht — CHANGES_REQUESTED** [review]

**Review by tuananht — APPROVED** [review]

**Comment by nguyen-tri-nhan** [comment]
https://trstringer.com/python-named-arguments/

**Pull request merged**